### PR TITLE
Update RadioToggle CSS

### PR DESCRIPTION
**Pull request by siegfried-ehret-sonarsource**
Use a pointer instead of the usual arrow to show that it is clickable.

### CODE REVIEW

**Comment by siegfried-ehret-sonarsource** [comment]
@gregaubert not sure if this is worth mentioning in the changelog...

**Review by philippe-perrin-sonarsource — APPROVED** [review]
I would add, does it even deserves to be a PR ? 😆

File: components/controls/RadioToggle.css [hunk 1]
```diff
@@ -52,6 +52,7 @@
   color: var(--darkBlue);
   height: calc(var(--controlHeight) - 2px);
   line-height: calc(var(--controlHeight) - 2px);
+  cursor: pointer;
```
Comment (gregaubert) [inline components/controls/RadioToggle.css:55]: Are you really sure it's not `pointer: cursor;` ? :trollface:

**Review by gregaubert — APPROVED** [review]
It looks ok, but given the huge number of changes and the complexity of the review we can never be 100% sure, so I'll just trust you on this one!

**Pull request merged**